ibis/backends/trino/registry.py

```diff
@@ -99,6 +102,48 @@ def _timestamp_truncate(t, op):
     return sa.func.date_trunc(precision, sa_arg)
 
 
+def _date_from_ymd(t, op):
+    ymdstr = sa.func.format(
+        '%04d-%02d-%02d',
+        t.translate(op.year),
+        t.translate(op.month),
+        t.translate(op.day),
+    )
+    return sa.func.date(ymdstr)
```

Comment:
```suggestion
    return sa.func.from_iso8601_date(ymdstr)
```

I know the function name is longer, but this way there's no guessing since we're already stipulating the format.